applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+- [An oracle network for Solana blockchain](https://github.com/fiit-ba/omniscient-blockchain-oracle)
+- [Tokengram social network for NFT holders](https://github.com/fiit-tp7-2023)
+- [On-chain representation of gaming assets](https://github.com/brano-hozza/meta-assets)
+
+
+### Team GitHub Profiles
+
+- https://github.com/Roman-24
+- https://github.com/xkostal
+- https://github.com/brano-hozza
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/romanbitarovsky/
+- https://www.linkedin.com/in/kristi%C3%A1n-ko%C5%A1%C5%A5%C3%A1l-9599a82b5/
+- https://www.linkedin.com/in/branislav-hozza-3b7234173/
+
+### Advisors
+Ecosystem developers who agreed to provide advisory help during the project implementation phase:
+- [Dudo50](https://github.com/dudo50), founder of ParaSpell (Interoperability solutions)
+
+## Development Status :open_book:
+
+This concept has already been implemented as a prototype dApp on an Ethereum-like chain, with its details discussed in two
+academic publications.
+The first is a [conference full paper](https://arxiv.org/pdf/2404.14074) presented at
+the [2024 IEEE International Conference on Blockchain and Cryptocurrency in Dublin](https://icbc2024.ieee-icbc.org/).
+The second is a [Bachelor's thesis](https://opac.crzp.sk/?fn=detailBiblioForm&sid=2D2C61E4341E5366536A1A3BFA9A) where
+this concept was originally developed and first implemented using Solidity smart contracts on Moonriver.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 6 months
+- **Full-Time Equivalent (FTE):**  2 FTE
+- **Total Costs:** 30,000 USD
+
+### Milestone 1 — Basic functionality
+
+- **Estimated duration:** 2.5 months
+- **FTE:**  2
+- **Costs:** 15,000 USD
+
+At first, we need to take pallet_nfts, take most core functions of NFS, and wrap these functionalities in our pallet. The reason to add pallet_nfts as a dependency and wrap the functions is to make usage of our pallet_nftaa easier and have all functionalities in one package for developers.
+
+Functionalities from pallet_nfts:
+- pallet::dispatchables::burn
+- pallet::dispatchables::buy_item
+- pallet::dispatchables::clear_attribute
+- pallet::dispatchables::clear_collection_metadata
+- pallet::dispatchables::clear_metadata
+- pallet::dispatchables::create
+- pallet::dispatchables::destroy
+- pallet::dispatchables::lock_collection
+- pallet::dispatchables::lock_item_properties
+- pallet::dispatchables::lock_item_transfer
+- pallet::dispatchables::mint
+- pallet::dispatchables::redeposit
+- pallet::dispatchables::set_attribute
+- pallet::dispatchables::set_collection_max_supply
+- pallet::dispatchables::set_collection_metadata
+- pallet::dispatchables::set_metadata
+- pallet::dispatchables::set_price
+- pallet::dispatchables::set_team
+- pallet::dispatchables::transfer
+- pallet::dispatchables::transfer_ownership
+- pallet::dispatchables::unlock_item_transfer
+- pallet::dispatchables::update_mint_settings
```

Comment:
Would you mind to describe these functions? Deducting meaning from function names can easily lead to misunderstandings. Thanks.

Reply:
Yes, in order not to re-do already made and to ensure compatibility we want to use the NFT pallet as a dependency. We are ready to maintain compatibility in future.